src/Commands/SetBrake.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SetBrake_H
+#define SetBrake_H
+
+#include "Commands/Command.h"
+#include "../Robot.h"
+
+class SetBrake: public Command
+{
+public:
+	SetBrake(Elevator::BrakeValue value);
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+
+private:
+	Elevator::BrakeValue mValue;
```

Comment:
Once more, adding Elevator.h to the includes give you a better idea of the dependencies of your class.